elearning-frontend/src/components/admin/Users.js

```diff
@@ -133,49 +133,57 @@ const Users = () => {
                             <tr key={user.id}>
                                 <td>{user.profile.firstname + " " + (user.profile.middlename !== null && user.profile.middlename !== "" ? user.profile.middlename + " " : "") + user.profile.lastname}</td>
                                 <td>
-                                    <div className='flex'>
-                                        <button onClick={(e) => {
-                                            e.preventDefault()
+                                    <div className='flex gap-2'>
+                                        <Button
+                                            text='EDIT'
+                                            type='button'
+                                            classes='bg-blue-500 hover:bg-blue-700'
+                                            onClick={(e) => {
+                                                e.preventDefault()
```

Comment:
create new function, dont add some logics here. it is so messy to look at.
This is a repeat comment, pls take note of this next time